src/Tickets/Seating/app/blockEditor/capacity-table.js

```diff
@@ -0,0 +1,42 @@
+import {select} from "@wordpress/data";
+import { storeName } from './store';
+import {addFilter} from "@wordpress/hooks";
+import { getLocalizedString } from '@tec/tickets/seating/utils';
+
+export function filterCapacityTableMappedProps(mappedProps) {
+	const hasSeats = select(storeName).isUsingAssignedSeating();
+	const layoutLocked = select(storeName).isLayoutLocked();
+
+	if ( ! hasSeats || ! layoutLocked ) {
```

Comment:
Collect into a !AND.

```suggestion
	if ( ! ( hasSeats && layoutLocked ) ) {
```